ids/mapping.json

```diff
@@ -17,6 +17,7 @@
   "nikhil": "whydonti",
   "om": "strider99",
   "prakash": "prakashchoudhary07",
+  "prem": "PremanandChowdhury",
```

Comment:
Since there's been a new entry `pranav`, you will have to shift your entry below his

Reply:
ok i'll pull the changes.
Also, will i push to develop or to main?

Reply:
I see that you've pushed from a branch called `PremanandChowdhury` from your fork. Check just below the title of the PR.

So in local, you should checkout to `PremanandChowdhury` branch, make changes, commit and push to `origin PremanandChowdhury` so that the new commits are added in this same PR.

The destination is RDS::main... So it'll be merged in there only... Doesn't matter from which branch of your fork it is coming.

Reply:
yeah i understand 🚀

Reply:
OMG 😱, It's **DONE** 👍🏼